### PR TITLE
fixed status bar, nav bar and device height

### PR DESCRIPTION
**Pull request by prklm10**
- We were not getting correct device height for all the devices.
- We were not getting correct nav bar height for all the devices whose nav bar is not visible by default settings. (e.g google pixel).
- We were getting wrong status bar for few of the new devices e.g pixel 7 pro

### CODE REVIEW

File: espresso/src/main/resources/deviceInfo.json [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+  },
+  "oppo reno3 pro": {
+    "statusBarHeight": 110
+  },
+  "samsung galaxy s23 ultra": {
+    "statusBarHeight": 75,
+    "navBarHeight": 135,
+    "deviceHeight": 2316
+  },
+  "realme  8": {
```
Comment (pankaj443) [inline espresso/src/main/resources/deviceInfo.json:73]: there seem to be 2 spaces here. also check anywhere else if its the case

File: espresso/src/main/resources/deviceInfo.json [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+  },
+  "samsung galaxy tab s7": {
+    "statusBarHeight": 51
+  },
+  "vivo v2111": {
+    "statusBarHeight": 60,
+    "navBarHeight": 79,
+    "deviceHeight": 1600
+  }
+}
```
Comment (pankaj443) [inline espresso/src/main/resources/deviceInfo.json:374]: new line

File: espresso/src/main/resources/deviceInfo.json (hunk 1 above)
Comment (prklm10) [reply espresso/src/main/resources/deviceInfo.json:73]: it seems to be present in devices.csv. check line `39036`.

File: espresso/src/main/resources/deviceInfo.json (hunk 2 above)
Comment (prklm10) [reply espresso/src/main/resources/deviceInfo.json:374]: my linter is removing extra line after saving 😿

**Review by pankaj443 — APPROVED** [review]

**Pull request merged**